squirrel/commands/watch.py

```diff
@@ -72,42 +69,44 @@ def get_daemon_pid() -> int:
 
 
 def daemon(wd, logger):
-    watch_flags = flags.CREATE | flags.MODIFY | flags.DELETE
 
     os.chdir(wd)
-    logger.debug('Adding inotify watches')
-    watches = (wd, )
-    i = INotify()
-    for watch in watches:
-        i.add_watch_recursive(watch, watch_flags)
-
-    engine = plugin.load_module()
-
+    watches = wd
+    project_files = Plugin.get_files(wd)
```

Comment:
I agree that It's better to have `get_files()` in a separate module but why have you chosen `plugin.py`?

Reply:
Haven't thought that much about it just read your comment and put it there since I was working on that file
```Python
# TODO: we might benefit from this becoming a plugin
def get_files(path):
```

Reply:
It's okay for now to keep it in `plugin.py`. If there's similar functionality in the future, we'll make a new module.